src/test-utils/createRender.d.ts

```diff
@@ -6,4 +6,4 @@ export interface RenderOptions {
 
 export default function createRender(
   options?: Partial<RenderOptions>,
-): typeof render & { cleanUp: Function };
```

Comment:
Is this a bug? `cleanUp` didn't actually exist in the JS.

Reply:
We don't need it with SSR.